libs/minizip/Makefile

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minizip
+PKG_VERSION:=2.9.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/nmoinvaz/minizip/archive/
+PKG_HASH:=f64b2d228a03673a8ec36a53402e2108437226fd9170d49b7f0c0bca94708b93
+
+PKG_MAINTAINER:=David Woodhouse <dwmw2@infradead.org>
+PKG_LICENSE:=Zlib
+PKG_LICENSE_FILES:=LICENSE
+
+CMAKE_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+CMAKE_OPTIONS:=-DINSTALL_INC_DIR=/usr/include/minizip -DMZ_BZIP2=OFF -DBUILD_SHARED_LIBS=yes
```

Comment:
for consistency, shared libs should be set to ON.

that inc dir is curious. Do you mean it's required to be installed on the host?

Reply:
No, it does indeed get installed in the staging dir, we want it in ${staging_dir}/usr/include/minizip not ${staging_dir}/usr/include which is the default. Partly because that's just cleaner, and partly because Domoticz has a bug where it doesn't actually honour the pkg-config at all and just *assumes* it's in `minizip/` (which I'm fixing in https://github.com/domoticz/domoticz/pull/4213 but which we don't need to wait for).